app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @review = Review.new(params_review)
+    @review.user_id = current_user.id
+    @review.video_id = params[:video_id]
+    @review.rating = params[:Rating]
```

Comment:
Yes, but in your controller specs you are using rating under review. Also @review.rating is same as mass assigning rating because you have set the setter method in your model. So you are resetting stuff on the fly. Anyway I suggest different way - `f.select :rating, options_for_select ....` will be good enough. If you have troubles implementing this, in solutions you should be able to follow it easily :)